packages/@o3r/core/schematics/rule-factories/fixture/index.ts

```diff
@@ -47,14 +47,12 @@ export function updateFixtureConfig(options: { projectName: string | null; testi
         return libs;
       }, []);
 
-      if (tsconfigCompilerOptions.paths) {
-        tsconfigCompilerOptions.paths = Object.keys(tsconfigCompilerOptions.paths).reduce((acc, p) => {
-          if (oldPaths.indexOf(p) === -1) {
-            acc[p] = tsconfigCompilerOptions.paths[p];
-          }
-          return acc;
-        }, {});
-      }
+      tsconfigCompilerOptions.paths = Object.keys(tsconfigCompilerOptions.paths || {}).reduce((acc, p) => {
```

Comment:
can use `Object.entries` to avoid access on line 52

Reply:
done